examples/grovers_lookup.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iomanip> // For setw
+#include <iostream> // For cout
+
+// "qfactory.hpp" pulls in all headers needed to create any type of "Qrack::QInterface."
+#include "qfactory.hpp"
+
+using namespace Qrack;
+
+int main()
+{
+    int i;
+
+    // ***Grover's search, to find a value in a lookup table***
+
+    // We search for 100, in the lookup table. All values in lookup table are 1 except a single match.
```

Comment:
This comment needs further explanation and expansion, with possibly some tabular diagrams.

Reply:
Added a better explanation, but I think we should keep it as simple as possible, while still communicating the general idea.